src/config.ini

```diff
@@ -39,14 +39,16 @@ rebuild = build --no-cache
 
 [mysql-backup-filter]
 
-typo3[] = "/^cachingframework_.*/i"
-typo3[] = "/^cf_.*/i"
-typo3[] = "/^cache_.*/i"
-typo3[] = "/^index_.*/i"
-typo3[] = "/^sys_log$/i"
-typo3[] = "/^sys_history$/i"
-typo3[] = "/^tx_extbase_cache.*/i"
-typo3[] = "/^tx_extensionmanager_.*/i"
+all[] = "/^.*$/i"
+
+typo3caching[] = "/^cachingframework_.*/i"
```

Comment:
My intension was to give this filter group a more meaningful name, because it's not "typo3" tables the user wants to exclude from transferring, it's only the "typo3 caching" tables. For other scenarios it might come in handy to just define "typo3news" tables or whatever.